colorbleed/plugins/maya/create/colorbleed_model.py

```diff
@@ -14,8 +12,12 @@ class CreateModel(avalon.maya.Creator):
     def __init__(self, *args, **kwargs):
         super(CreateModel, self).__init__(*args, **kwargs)
 
-        data = {"writeColorSets": False,  # Vertex colors with the geometry.
-                "attr": "",  # Add options for custom attributes
-                "attrPrefix": ""}
+        # Vertex colors with the geometry
```

Comment:
This change for the existing is solely so that the order is preserved correctly for the attributes on the node. It's not required for this feature.